tools/development/cx_freeze.py

```diff
@@ -203,7 +196,7 @@
 setup(
 	name='KingPhisher',
 	author='SecureState',
-	version=version.distutils_version,
+	version=re.sub("[^0-9.]", "", version.distutils_version),
```

Comment:
This could be in accurate. By simply removing the letters, the version number isn't always the same value. Take 1.1.0b7 (beta 7) for example would be 1.1.07 which if you convert those to digits for a comparison would be 1.1.7.

Please use version.version_info instead.